src/middleSection.js

```diff
@@ -89,8 +96,32 @@ const changeMiddleSegment = (cities) => {
     middleContainer.appendChild(card)
     document.getElementsByClassName('items-scroll')[i].style.backgroundImage = "url('/Assets/HTML&CSS/Icons_for_cities/" + value.cityName.toLowerCase() + ".svg')"
     i = i + 1
+    scrollEventPassing(card, document.querySelector('.items-container'))
   })
   updateScrollButtonsVisibility()
 }
 
-export const middleSection = { updateScrollButtonsVisibility, addCardsbySpinner, scrollItems, selectIcon, changeMiddleSegment, filterMiddleSegment }
+/**
+Method to scroll Middle section both Horizontally and Vertically
+ * @param {object} target - target element
+ * @param {object} targetContainer - underlying container
+ */
+function scrollEventPassing (target, targetContainer) {
+  target.addEventListener('wheel', (evt) => {
+    evt.preventDefault()
+
+    if (evt.deltaY !== 0) {
+      window.scrollBy({
+
+        top: evt.deltaY
+
+      })
+    }
+
+    if (evt.deltaX !== 0) {
+      targetContainer.scrollLeft += evt.deltaX
+    }
+  })
+}
+scrollEventPassing(document.querySelector('.items-overlay'), document.querySelector('.items-container'))
```

Comment:
If we're calling the function by default, we can check if it can be used with IIFE

Reply:
IIFE functions cant be called outside